lib/main/utils/toast_manager.dart

```diff
@@ -33,4 +42,26 @@ class MessageToastUtils {
       return null;
     }
   }
+
+  void showMessageFailure(Failure failure) {
+    if (currentContext == null || currentOverlayContext == null) {
+      logError('ToastManager::showMessageFailure: Context is null');
+      return;
+    }
+
+    String? message;
+
+    if (failure is GetSessionFailure) {
+      message = getMessageByException(currentContext!, failure.exception)
+        ?? AppLocalizations.of(currentContext!).unknownError;
+    } else if (failure is EmptySpamFolderFailure) {
+      message = AppLocalizations.of(currentContext!).emptySpamFolderFailed;
+    } else if (failure is MoveMultipleEmailToMailboxFailure) {
+      message = AppLocalizations.of(currentContext!).markAsSpamFailed;
```

Comment:
I see this failure also in `unspam`, `move` and `undo move` actions. This message is not correct for all of these cases

Reply:
We do not show messages to them.

Reply:
I suggest we have a dedicated message for markAsSpam failure

Reply:
Done